eng/scripts/analyze-step-helpers.ps1

```diff
@@ -0,0 +1,100 @@
+Write-Host "$PSScriptRoot"
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+
+<#
+.DESCRIPTION
+    Create .gosource APIVIew artifact for go
+.PARAMETER ServiceDirectory
+    Thee name of the ServiceDirectory
+.PARAMETER OutputDirectory
+    Base output Directory path for the generated gosource artifacts
+.PARAMETER DirectoryToPublish
+    Directory containing all artifacts to be publisehd to the pipeline
+#>
+function New-GoArtifacts {
```

Comment:
Can you please rename this function to `Create-ApiView-Artifact` to be more specific. Go does not have any artifact and this generic name will be misleading.